x/rollapp/keeper/liveness.go

```diff
@@ -59,12 +58,13 @@ func (k Keeper) CheckLiveness(ctx sdk.Context) {
 
 // HandleLivenessEvent will slash or jail and then schedule a new event in the future.
 func (k Keeper) HandleLivenessEvent(ctx sdk.Context, e types.LivenessEvent) error {
-	err := k.sequencerKeeper.SlashLiveness(ctx, e.RollappId)
+	err := k.SequencerK.SlashLiveness(ctx, e.RollappId)
 	if err != nil {
 		return errorsmod.Wrap(err, "slash liveness")
 	}
 
 	ra := k.MustGetRollapp(ctx, e.RollappId)
+	k.DelLivenessEvents(ctx, e.HubHeight, e.RollappId)
```

Comment:
@omritoptix fixes a horrible bug which I think would have caused us to slash every block instead of on interval

Reply:
ha that explains the fast slashing of in the previous simulation!

Reply:
yeah sorry had another look but the bug is causing old events to not be deleted (but it doesn't re-read them)
So no impact on users, but it is a storage leak